NMS-14182: Clean up Alarmd doc formatting (#4557)
OpenNMS/opennms: docs/modules/operation/pages/alarms/alarm-notes.adoc

Alarm notes let you add two types of notes on an existing alarm or alarm class:

* Sticky Memo: A user-defined note for a specific instance of an alarm.
Deleting the alarm will also delete the sticky memo.

* Journal Memo: A user-defined note for a whole class of alarms based on the resolved reduction key.
The journal memo will be shown for all alarms matching a specific reduction key.

> Deleting an alarm doesn't remove the journal memo, they can be removed by pressing the "Clear" button on an alarm with the existing journal memo.

Reviewer: ```suggestion
Deleting an alarm doesn't remove the journal memo.
Press the "Clear" button on an alarm with the existing journal memo to remove it.
```

> If an alarm has a sticky and/or a journal memo it is indicated with icons on the "Alarm list Summary" and "Alarm List Detail".

Reviewer: ```suggestion
Icons on the "Alarm list Summary" and "Alarm List Detail" indicate that an alarm has a sticky and/or a journal memo.
```

Author: What about:
```suggestion
Icons on the "Alarm list Summary" and "Alarm List Detail" indicate if an alarm has sticky and/or journal memos.
```
